extensions/control-plane/api/management-api/contract-agreement-api/src/test/java/org/eclipse/edc/connector/api/management/contractagreement/ContractAgreementNewApiControllerTest.java

```diff
@@ -52,15 +51,21 @@
 class ContractAgreementNewApiControllerTest extends RestControllerTestBase {
 
     private final ContractAgreementService service = mock(ContractAgreementService.class);
-    private final JsonLd jsonLdService = new TitaniumJsonLd(monitor);
+    private final JsonLd jsonLdService = mock(JsonLd.class);
```

Comment:
nit: in all the other API tests we've initialized this with the actual `TitaniumJsonLd` class to avoid all the mock setups.

if you want to keep the mock, i would do the `when(jsonLdService)....` in the `@BeforeEach`

Reply:
Restored `TitaniumJsonLd`